### PR TITLE
Update IDP Discovery Policy with Multi IDP support

### PR DESCRIPTION
**Pull request by romantkachuk-okta**
## Description:
- Add Multi IdP support description brought together with OIE

### Resolves:

* [OKTA-385366](https://oktainc.atlassian.net/browse/OKTA-385366)

### CODE REVIEW

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -1457,3 +1458,51 @@ The following conditions may be applied to IdP Discovery Policy:
...
+
+| Property                           | Description             | Data Type | Required |
+| ---                                | ---                     | ---       | ---      |
+| id                                 | Provider `id` in Okta   | String    | Yes      |
+| type                               | Provider type           | `OKTA`, `AgentlessDSSO`, `IWA`, `X509`, `SAML2`, `OIDC`, `APPLE`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT` | Yes |
+| name <ApiLifecycle access="ie" />  | Provider `name` in Okta | String    | No       |
+
+##### Limitations
+
+* Single rule can contain with up to 10 providers
```
[the remark this answers is not in the document]
Comment (romantkachuk-okta) [reply packages/@okta/vuepress-site/docs/reference/api/policy/index.md]: I've rephrased it a little bit. Meant Policy Action

**Comment by romantkachuk-okta** [comment]
@jakubvul-okta please review updates, thanks!

**Review by jakubvul-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 2]
```diff
@@ -1457,3 +1458,52 @@ The following conditions may be applied to IdP Discovery Policy:
...
+Multiple IdP instances can be defined in a single Policy Action. It allows User to choose a Provider to sign in with.
+
+##### Provider object
+
+> **Note:** IdP types `OKTA`, `AgentlessDSSO`, and `IWA` don't require an `id`.
+
+| Property                           | Description             | Data Type | Required |
+| ---                                | ---                     | ---       | ---      |
+| id                                 | Provider `id` in Okta   | String    | Yes      |
+| type                               | Provider type           | `OKTA`, `AgentlessDSSO`, `IWA`, `X509`, `SAML2`, `OIDC`, `APPLE`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT` | Yes |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:1476]:
```suggestion
| type                               | Provider type. Possible values: `OKTA`, `AgentlessDSSO`, `IWA`, `X509`, `SAML2`, `OIDC`, `APPLE`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT`  | String | Yes |
```

**Review by jakubvul-okta — APPROVED** [review]

**Review by jameszhang-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 3]
```diff
@@ -1457,3 +1458,52 @@ The following conditions may be applied to IdP Discovery Policy:
     }
   }
 ```
+
+#### Policy Action with multiple IdP instances
+<ApiLifecycle access="ie" />
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+Multiple IdP instances can be defined in a single Policy Action. It allows User to choose a Provider to sign in with.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:1467]:
```suggestion
Multiple IdP instances can be defined in a single Policy Action. This allows users to choose a Provider when they sign in.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md (hunk 2 above)
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:1476]:
```suggestion
| type                               | Provider type. Possible values: `OKTA`, `AgentlessDSSO`, `IWA`, `X509`, `SAML2`, `OIDC`, `APPLE`, `FACEBOOK`, `GOOGLE`, `LINKEDIN`, `MICROSOFT`           | String | Yes |
```

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**